Library/Tests/TestHelpers/TestCase.swift

```diff
@@ -23,11 +23,16 @@ internal class TestCase: FBSnapshotTestCase {
 
   override func setUp() {
     super.setUp()
+
+    var calendar = Calendar(identifier: .gregorian)
+    calendar.timeZone = TimeZone(identifier: "GMT")!
```

Comment:
previously tests were using `Calendar.current` which could mean people running tests with their computer set to a different calendar could get failing tests.

Of course, Im not sure who uses different calendars on their computers... seems wild...